.github/workflows/pr.yml

```diff
@@ -6,16 +6,31 @@ on:
 
 jobs:
   test:
-    strategy:
-      matrix:
-        go-version: [1.15.x, 1.16.x]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: crc
+          POSTGRES_USER: crc
+          POSTGRES_PASSWORD: crc
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: ${{ matrix.go-version }}
+        go-version: 1.16
+    - name: Migrate DB
```

Comment:
@tahmidefaz would it also make sense to seed the DB after migrating it?

Reply:
@aleccohan I don't think it's needed, since all the data that a test should look for will need to be created by the tests itself. We won't need to read anything from the seeded data.

Reply:
cool!  Yup that makes sense!